### PR TITLE
derpfest: add brazilian portuguese translation

### PR DESCRIPTION
**Pull request by jeangraff30**

### CODE REVIEW

**Comment by jeangraff30** [comment]
From what I see I didn't have the values-pt-rBR folder, so I checked the other folders, and I believe this folder is right

**Comment by poad42** [comment]
Might wanna edit those copyright headers

**Comment by idoybh** [comment]
> Might wanna edit those copyright headers

True

File: res/values-pt/aosip_strings.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+     Copyright (C) 2020 The PixelDust Project
```
Comment (idoybh) [inline res/values-pt/aosip_strings.xml]:
```suggestion
     Copyright (C) 2020 DerpFest
```

**Review by idoybh — CHANGES_REQUESTED** [review]

**Review by idoybh — APPROVED** [review]

**Review by idoybh — APPROVED** [review]

**Pull request merged**